### PR TITLE
placement: add some benchmark cases for FitRegion

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### What problem does this PR solve?

add some benchmark cases for FitRegion

### Release note

```release-note
None
```

### CODE REVIEW

File: server/schedule/placement/fit_region_test.go [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+		FitRegion(storesSet, region, rules)
+	}
+}
+
+func BenchmarkFitRegionMoreVotersSplitRules(b *testing.B) {
+	i := 0
+	region := mockRegion(5, 0)
+	rules := []*Rule{
+		{
+			GroupID:        "pd",
```
Comment (rleungx) [inline server/schedule/placement/fit_region_test.go:245]: Is it on purpose to use the same rule?

Comment (Yisaer) [reply server/schedule/placement/fit_region_test.go:245]: yes, this is used to comparing the cpu resource consuming between less rules and more rules.

Comment (rleungx) [reply server/schedule/placement/fit_region_test.go:245]: Then how about using a variable?

Comment (Yisaer) [reply server/schedule/placement/fit_region_test.go:226]: updated.

File: server/schedule/placement/fit_region_test.go [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+			ID:             "default",
+			Role:           Voter,
+			Count:          3,
+			LocationLabels: []string{},
+		},
+	}
+	storesSet := newMockStoresSet(100)
+	b.ResetTimer()
+	for i < b.N {
```
Comment (rleungx) [inline server/schedule/placement/fit_region_test.go:119]: use `for i := 0; i < b.N; i++`

**Pull request merged**